stl/inc/memory

```diff
@@ -308,6 +308,20 @@ _NoThrowFwdIt uninitialized_fill_n(_NoThrowFwdIt _First, const _Diff _Count_raw,
 }
 #endif // _HAS_IF_CONSTEXPR
 
+#if _HAS_CXX20
+template <class _Ty, class... _Args_pack>

```

Comment:
There is already a similar implementation of that function in `<xmemory>`

https://github.com/microsoft/STL/blob/ed3cbf36416a385828e7a5987ca52cb42882d84b/stl/inc/xmemory#L200-L205

Looking at it closely it seems you should go with the definition of `voidify` in P0784R7 which is 
```cpp
    template<class T>
    constexpr void* voidify(T& ptr) noexcept {
        return const_cast<void*>(static_cast<const volatile void*>(addressof(ptr)));
    } 
```

And `construct_at` has the effect of
```cpp
    return ::new (voidify(*location)) T(std::forward<Args>(args)...);
```

So this should read
```cpp
    return ::new (const_cast<void*>(static_cast<const volatile void*>(_Location) _Ty(_STD forward<_Args_pack>(_Args)...);
```

Note that in `_Construct_at` the pack type is written as `_Types` so we should be consistent

Reply:
You're right. I will change it.